x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -771,6 +823,16 @@ export class ExecutionHandler<
         }
 
         const actionGroup = this.getActionGroup(alert);
+        const previousActionGroup = alert.getLastScheduledActions()?.group;
+        console.log(`alert action group ${actionGroup}`);
```

Comment:
Why log to console from the alerts? Ideally, the task runner should use the kibana logging system